### ISSUE TITLE
Southwest Airlines App Blank Pages

### ISSUE THREAD

**Issue by seniorm0ment** [GrapheneOS/os_issue_tracker#2731]
The Southwest Airlines app, I can login no issues.
However a handful of important pages are just blank white. Some pages display no issues.

Some of the ones I show issue with are flight status, reservation, offers, travel funds, and more. 

Pixel 8 Pro
Version: 2023111500
Google Services enabled
Exploit Compatibility enabled on app

**Comment by thestinger** [comment]
You need to provide more information such as where you installed it from and whether you changed any relevant configuration.

**Comment by seniorm0ment** [comment]
Apologies
Installed through Aurora Store
What other relevant configs do you need? I believe I provided all that at the bottom.

Google Services enabled
Exploit Compatibility enabled on app

**Comment by thestinger** [comment]
> Installed through Aurora Store

This could be the issue. Did you use their Session installer mode? Native installer is broken.

**Comment by seniorm0ment** [comment]
It is set to use Session installer

**Comment by pdfruth** [comment]
Albeit not exactly the same, I was having a similar problem.
After installing the Southwest app, upon opening the app, the app would just hang after displaying the initial splash screen, showing the SW logo on an all-blue background.

I've installed the Southwest Airlines app version 10.12.0 on my Pixel 4 XL running GrapheneOS.
I installed the app via Aurora Store anonymous session installer.

Here is what I did to resolve the issue on my phone;
Go to app settings, scroll down to Advanced section and turn on the "Exploit protection compatibility mode" toggle.
Note: This seems to have also turned on "DISABLE_HARDENED_MALLOC" and "ENABLE_COMPAT_VA_39_BIT" toggles in the "Configure hardening" section.

Hopefully this might help others.
